com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -292,40 +300,99 @@ inline void InitializeBRDFData(half3 albedo, half metallic, half3 specular, half
     half reflectivity = 1.0 - oneMinusReflectivity;
 
     outBRDFData.diffuse = albedo * oneMinusReflectivity;
-    outBRDFData.specular = lerp(kDieletricSpec.rgb, albedo, metallic);
+    outBRDFData.specular = lerp(kDielectricSpec.rgb, albedo, metallic);
 #endif
 
-    outBRDFData.grazingTerm = saturate(smoothness + reflectivity);
+    outBRDFData.grazingTerm         = saturate(smoothness + reflectivity);
     outBRDFData.perceptualRoughness = PerceptualSmoothnessToPerceptualRoughness(smoothness);
-    outBRDFData.roughness = max(PerceptualRoughnessToRoughness(outBRDFData.perceptualRoughness), HALF_MIN);
-    outBRDFData.roughness2 = outBRDFData.roughness * outBRDFData.roughness;
-
-    outBRDFData.normalizationTerm = outBRDFData.roughness * 4.0h + 2.0h;
-    outBRDFData.roughness2MinusOne = outBRDFData.roughness2 - 1.0h;
+    outBRDFData.roughness           = max(PerceptualRoughnessToRoughness(outBRDFData.perceptualRoughness), HALF_MIN_SQRT);
+    outBRDFData.roughness2          = max(outBRDFData.roughness * outBRDFData.roughness, HALF_MIN);
+    outBRDFData.normalizationTerm   = outBRDFData.roughness * 4.0h + 2.0h;
+    outBRDFData.roughness2MinusOne  = outBRDFData.roughness2 - 1.0h;
 
 #ifdef _ALPHAPREMULTIPLY_ON
     outBRDFData.diffuse *= alpha;
     alpha = alpha * oneMinusReflectivity + reflectivity;
 #endif
 }
 
+// Approximation of ConvertF0ForAirInterfaceToF0ForClearCoat15
```

Comment:
this should be move to core as well: ConvertF0ForAirInterfaceToF0ForClearCoat15

Reply:
It should be now.